freeipa/src/main/java/com/sequenceiq/freeipa/flow/freeipa/verticalscale/actions/AbstractFreeIPAVerticalScaleAction.java

```diff
@@ -0,0 +1,95 @@
+package com.sequenceiq.freeipa.flow.freeipa.verticalscale.actions;
+
+import static com.sequenceiq.cloudbreak.cloud.model.AvailabilityZone.availabilityZone;
+import static com.sequenceiq.cloudbreak.cloud.model.Location.location;
+import static com.sequenceiq.cloudbreak.cloud.model.Region.region;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+import javax.inject.Inject;
+
+import org.springframework.statemachine.StateContext;
+
+import com.sequenceiq.cloudbreak.cloud.context.CloudContext;
+import com.sequenceiq.cloudbreak.cloud.model.CloudCredential;
+import com.sequenceiq.cloudbreak.cloud.model.CloudStack;
+import com.sequenceiq.cloudbreak.cloud.model.Location;
+import com.sequenceiq.cloudbreak.common.event.Payload;
+import com.sequenceiq.cloudbreak.logger.MDCBuilder;
+import com.sequenceiq.flow.core.FlowParameters;
+import com.sequenceiq.freeipa.converter.cloud.CredentialToCloudCredentialConverter;
+import com.sequenceiq.freeipa.converter.cloud.InstanceMetaDataToCloudInstanceConverter;
+import com.sequenceiq.freeipa.converter.cloud.ResourceToCloudResourceConverter;
+import com.sequenceiq.freeipa.converter.cloud.StackToCloudStackConverter;
+import com.sequenceiq.freeipa.entity.Stack;
+import com.sequenceiq.freeipa.flow.chain.AbstractCommonChainAction;
+import com.sequenceiq.freeipa.flow.freeipa.verticalscale.FreeIPAVerticalScaleState;
+import com.sequenceiq.freeipa.flow.freeipa.verticalscale.event.FreeIPAVerticalScaleEvent;
+import com.sequenceiq.freeipa.flow.freeipa.verticalscale.event.FreeIPAVerticalScaleFailureEvent;
+import com.sequenceiq.freeipa.flow.stack.StackContext;
+import com.sequenceiq.freeipa.service.CredentialService;
+import com.sequenceiq.freeipa.service.resource.ResourceService;
+import com.sequenceiq.freeipa.service.stack.StackService;
+
+public abstract class AbstractFreeIPAVerticalScaleAction<P extends Payload>
```

Comment:
please rename everything like `FreeIpa`, we are trying to be conistent with this, I know there are some changes already in place with `FreeIPA` and `Freeipa`